pkg/controller/defaults/resources.go

```diff
@@ -59,14 +59,29 @@ var (
 				corev1.ResourceMemory: resource.MustParse("3Gi"),
 			},
 		},
-		"noobaa": corev1.ResourceRequirements{
+		"noobaa-core": corev1.ResourceRequirements{
 			Requests: corev1.ResourceList{
-				corev1.ResourceCPU:    resource.MustParse("4"),
-				corev1.ResourceMemory: resource.MustParse("8Gi"),
+				corev1.ResourceCPU:    resource.MustParse("2"),
+				corev1.ResourceMemory: resource.MustParse("4Gi"),
 			},
 			Limits: corev1.ResourceList{
-				corev1.ResourceCPU:    resource.MustParse("4"),
-				corev1.ResourceMemory: resource.MustParse("8Gi"),
+				corev1.ResourceCPU:    resource.MustParse("2"),
+				corev1.ResourceMemory: resource.MustParse("4Gi"),
+			},
+		},
+		"noobaa-db": corev1.ResourceRequirements{
+			Requests: corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("2"),
+				corev1.ResourceMemory: resource.MustParse("4Gi"),
+			},
+			Limits: corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("2"),
+				corev1.ResourceMemory: resource.MustParse("4Gi"),
+			},
+		},
+		"noobaa-db-vol": corev1.ResourceRequirements{
+			Requests: corev1.ResourceList{
+				corev1.ResourceStorage: resource.MustParse("50Gi"),
```

Comment:
This will likely cause issues with ocs-ci and the ocs-operator functional tests.  In order for those test suites to run on the smaller m4.xlarge instances, we remove the resource requirements for those StorageClusters. This means that noobaa-db-vol won't be set.

Reply:
looking closer at how the defaulting works, unless someone explicitly sets `noobaa-db-vol` with empty request/limits lists, then we should be fine.

i think what you have will work as expected. sorry for the confusion.

Reply:
thanks!